README.md

```diff
@@ -26,7 +26,94 @@ Estimating the geographical range of a species from sparse observations is a cha
 ```
 
 #### Data Download and Preparation
-Instructions for downloading the data in `data/README.md`.
+# Instructions for Data Preparation
+After following these instructions, the `data` directory should have the following structure:
```

Comment:
this directory explanation section is not readable, can we move it to after the instructions of how to setup and make it clear it's just an explanation of how the folder is structured. 

If we are going to put all the readmes together, we should try to make them readable for the next user.

Reply:
new changes should fixed it